builtin/game/item.lua

```diff
@@ -724,7 +726,7 @@ core.craftitemdef_default = {
 	inventory_image = "",
 	wield_image = "",
 	wield_scale = {x=1,y=1,z=1},
-	stack_max = 99,
+	stack_max = stacksize_default_item,
```

Comment:
The fact that this is user-changeable should be stated in lua_api.txt too.

Reply:
Yes, please update line 6729 of lua_api to say that the value may be configured by default.

```
-- The default value of 99 may be configured by users using the setting "stacksize_default_item"
stack_max = 99,
```